src/models/jwtPermissions/customErrors/notSufficientPermission.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const { NotAuthorized } = require('./notAuthorized');
+
+class NotSufficientPermissions extends NotAuthorized {
```

Comment:
If your token is valid, but you have insufficient rights, its 403 (Forbidden).

Altough it seems similar, it is important to distinguish because 401 usually triggers token refreshing, while 403 not.

Reply:
done